Documentation/project-docs/clr-configuration-knobs.md

```diff
@@ -3,7 +3,7 @@
 
 This Document is machine-generated from commit 65f7881 on 08/15/15. It might be out of date.
 
-When using these configurations from environment variables, the variable need to have `ComPlus_` prefix in its name. e.g. To set DumpJittedMethods to 1, add `ComPlus_DumpJittedMethods=1` to envvars.
+When using these configurations from environment variables, the variable need to have `COMPlus_` prefix in its name. e.g. To set DumpJittedMethods to 1, add `COMPlus_DumpJittedMethods=1` to envvars.
```

Comment:
As the top of this file indicates, this document is apparently machine generated. Changes here could be overwritten if the document is re-generated. Rather than updating this file, wouldn't it make more sense to update the places where this file is getting these values?